aip/general/0132.md

```diff
@@ -160,11 +160,8 @@ always possible to add ordering later, but removing it is a breaking change.
 ### Filtering
 
 List methods **may** allow clients to specify filters; if they do, the request
-message **should** contain a `string filter` field.
-
-<!-- TODO(#85): Add a reference to a filtering AIP once
-                that AIP is written.
-  -->
+message **should** contain a `string filter` field. Filtering is described in
+more detail in AIP-160.
```

Comment:
Does this automagically become a clickable link?

Reply:
Yes.